blueprints/disruption-budgets/README.md

```diff
@@ -0,0 +1,208 @@
+# Karpenter Blueprint: NodePool Disruption Budgets
+
+## Purpose
+
+A Karpenter’s NodePool disruption budgets can be used to rate limit Karpenter’s voluntary disruptions, if no budget is set their is a default of 10% set. When calculating if a budget will block nodes from disruption, Karpenter checks if the number of nodes being deleted is greater than the number of allowed disruptions. Budgets take into consideration voluntary disruptions through expiration, drift, emptiness and consolidation. If there are multiple budgets defined in the NodePool, Karpenter will honour the most restrictive of the budgets.
```

Comment:
I think we may require an entry to explain why disruption controls are needed and be prescriptive on the use case. I'd suggest something like:

"Actions like consolidation, drift detection and ExpireAfter, allow users to optimize for cost in the case of consolidation, keep up with the latest security patches and gold configuration, or ensure governance best practices, like refreshing instances every N days.  This actions cause, as a trade-off, some level of disruption in the cluster caused by expected causes.  To control the trade-off between, for example, being on the latest AMI (drift detection) and nodes restarting when that happen we can use disruption controls and set up disruption budgets. Disruption budgets can help us set a maintenance window in the example above to allow for drift to happen at low use hours for example.